### PR TITLE
Clarify that min can also be a string

### PR DESCRIPTION
**Pull request by tillydray**
#### Summary

`min` and `max` can be both a string or a number... right? I don't know how I would represent an iso8601 date as a number. In fact, the examples show all of these are strings _except_ `number` and `range`

#### Motivation

fix an (assumed) error

#### Supporting details

From [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/max):

Input type | Example | Example
-- | -- | --
date | yyyy-mm-dd | `<input type="date" min="2019-12-25" step="1">`
month | yyyy-mm | `<input type="month" min="2019-12" step="12">`
week | yyyy-W## | `<input type="week" min="2019-W23" step="">`
time | hh:mm | `<input type="time" min="09:00" step="900">`
datetime-local | yyyy-mm-ddThh:mm | `<input type="datetime-local" min="2019-12-25T19:30">`
number | `<number>` | `<input type="number" min="0" step="5" max="100">`
range | `<number>` | `<input type="range" min="60" step="5" max="100">`

#### Related issues

n/a

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

similar PR: https://github.com/mdn/content/pull/14481

### CODE REVIEW

File: files/en-us/web/html/attributes/min/index.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ spec-urls: https://html.spec.whatwg.org/multipage/input.html#the-min-and-max-att
 
 The **`min`** attribute defines the minimum value that is acceptable and valid for the input containing the attribute. If the [`value`](/en-US/docs/Web/HTML/Element/input#attr-value) of the element is less than this, the element fails [constraint validation](/en-US/docs/Web/Guide/HTML/Constraint_validation). This value must be less than or equal to the value of the `max` attribute. If a value is specified for `min` that isn't a valid number, the input has no minimum value.
```
Comment (hamishwillee) [inline files/en-us/web/html/attributes/min/index.md:16]:
```suggestion
The **`min`** attribute defines the minimum value that is acceptable and valid for the input containing the attribute. If the [`value`](/en-US/docs/Web/HTML/Element/input#attr-value) of the element is less than this, the element fails [constraint validation](/en-US/docs/Web/Guide/HTML/Constraint_validation). This value must be less than or equal to the value of the `max` attribute.
If a value is specified for `min` that can't be converted to a valid number, the input has no minimum value.
```

File: files/en-us/web/html/attributes/min/index.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ spec-urls: https://html.spec.whatwg.org/multipage/input.html#the-min-and-max-att
...
 
-Valid for the numeric input types, including the {{HTMLElement("input/date", "date")}}, {{HTMLElement("input/month", "month")}}, {{HTMLElement("input/week", "week")}}, {{HTMLElement("input/time", "time")}}, {{HTMLElement("input/datetime-local", "datetime-local")}}, {{HTMLElement("input/number", "number")}} and {{HTMLElement("input/range", "range")}} types, and the {{htmlelement('meter')}} element, the `min` attribute is a number that specifies the most negative value a form control to be considered valid.
+Valid for the numeric input types, including the {{HTMLElement("input/date", "date")}}, {{HTMLElement("input/month", "month")}}, {{HTMLElement("input/week", "week")}}, {{HTMLElement("input/time", "time")}}, {{HTMLElement("input/datetime-local", "datetime-local")}}, {{HTMLElement("input/number", "number")}} and {{HTMLElement("input/range", "range")}} types, and the {{htmlelement('meter')}} element, the `min` attribute is a string or number that specifies the most negative value a form control to be considered valid.
```
Comment (hamishwillee) [inline files/en-us/web/html/attributes/min/index.md:18]:
```suggestion
It is valid for the input types including: {{HTMLElement("input/date", "date")}}, {{HTMLElement("input/month", "month")}}, {{HTMLElement("input/week", "week")}}, {{HTMLElement("input/time", "time")}}, {{HTMLElement("input/datetime-local", "datetime-local")}}, {{HTMLElement("input/number", "number")}} and {{HTMLElement("input/range", "range")}} types, and the {{htmlelement('meter')}} element.
```

**Comment by hamishwillee** [comment]
@JasonMFry I think you're right in that values are not strictly numeric: they are expressed as strings, and according to the spec are converted to numbers for the range comparison. If the value is invalid/can't be converted and there is no default minimum then the input will have no minimum.

That said, I think the problem here is that we're referring to these things as numeric input types, which is what is making you want to clarify the value can be a string. I don't think that is necessary/helpful. So I've proposed an alternative above which keeps these as values.

@ericwbailey is a the expert though, so will doubtless provide final review.

**Comment by JasonMFry** [comment]
> I think the problem here is that we're referring to these things as numeric input types, which is what is making you want to clarify the value can be a string. I don't think that is necessary/helpful. 

@hamishwillee I was pretty sure I had something wrong here, but was hoping to get it straightened out by others such as yourself. Thanks!

**Comment by hamishwillee** [comment]
@JasonMFry Sadly I'm not an expert either, which is why I'm deferring to Eric or @estelle for approval. Both of our suggestions are better than what was there before, so thanks for posting.

**Review by estelle — APPROVED** [review]

**Pull request merged**